examples/02_decoding/plot_haxby_decoder.py

```diff
@@ -0,0 +1,58 @@
+"""
+Decoding with decoder metaestimator: face vs house recognition
+==============================================================
+
+Here is a simple example of decoding with a the decoder metaestimator,
+reproducing a face vs house discrimination task on the study "Distributed and
+Overlapping Representations of Faces and Objects in Ventral Temporal Cortex"
+(Science 2001).
+
```

Comment:
Sorry, I was not clear enough regarding references part. You can have a look at example ```plot_canica_resting_state.py``` for example pointing a reference.